.gitignore

```diff
@@ -6,3 +6,5 @@ templates/*
 
 !/templates/config.yaml
 !/templates/default.mustache
+
+.venv
```

Comment:
Accepting for now, but just a tip: you can have your own global `.gitignore`.

In you `.gitconfig`:

```
[core]
  excludesfile = ~/.gitignore
```
Then add `.venv` to that file :)

Reply:
This is helpful, thanks a lot!